src/lib/helpers/flag.ts

```diff
@@ -36,3 +36,23 @@ export const dirPath = Flags.custom<DirContext>({
     return { abspath, exists };
   },
 });
+
+/*
+ * Takes a flag input that's supposed to be a JSON string and validates it.
+ */
+export const jsonStr = Flags.custom<string>({
+  parse: async (input: string) => {
+    try {
+      const data = JSON.parse(input);
+      return data;
+    } catch {
+      throw new Error(`${input} is not a valid JSON string.`);
+    }
+  },
+});
+
+export const commaSeparatedArray = Flags.custom<string[]>({
+  parse: async (input: string) => {
+    return input.split(",");
```

Comment:
```suggestion
    return input.split(",").filter(x => x);
```
I'm thinking we just filter out any empty string e.g. `"foo,,bar"` -> `["foo", "bar"]` instead of `["foo", "", "bar"]`